compiler/test/dotty/tools/repl/TabcompleteTests.scala

```diff
@@ -126,6 +126,10 @@ class TabcompleteTests extends ReplTest {
     assertEquals(List("Predef"), tabComplete("object Foo { opaque type T = Pre"))
   }
 
+  @Test def i6361 = initially {
+    assertEquals(Nil, tabComplete("object foo { given bar: Int = 10 }; import foo.*"))
```

Comment:
Shouldn't that string end with `; ba` to test the faulty completion?

Reply:
:facepalm: thanks for catching that